README.md

```diff
@@ -641,29 +695,6 @@ you can then use mqtt.js in the browser with the same api than node's one.
 
 Your broker should accept websocket connection (see [MQTT over Websockets](https://github.com/mcollina/mosca/wiki/MQTT-over-Websockets) to setup [Mosca](http://mcollina.github.io/mosca/)).
 
-<a name="signedurls"></a>
-### Signed WebSocket Urls
-
-If you need to sign an url, for example for [AWS IoT](http://docs.aws.amazon.com/iot/latest/developerguide/protocols.html#mqtt-ws),
-then you can pass in a `transformWsUrl` function to the <a href="#connect"><code>mqtt.<b>connect()</b></code></a> options
-This is needed because signed urls have an expiry and eventually upon reconnects, a new signed url needs to be created:
-
-```js
-// This module doesn't actually exist, just an example
-var awsIotUrlSigner = require('awsIotUrlSigner')
-mqtt.connect('wss://a2ukbzaqo9vbpb.iot.ap-southeast-1.amazonaws.com/mqtt', {
-  transformWsUrl: function (url, options, client) {
-    // It's possible to inspect some state on options(pre parsed url components)
-    // and the client (reconnect state etc)
-    return awsIotUrlSigner(url)
-  }
-})
-
-// Now every time a new WebSocket connection is opened (hopefully not that
```

Comment:
can you add this line to your explanation, i think it is a nice clear and concise way of explaining what the code snippet is doing.

Reply:
sounds good, added. Do you want to still mention "awsIotUrlSigner"? I removed the specific aws mention since there could be other cloud providers that do the same thing. But, how this came up for me in the first place is that I'm using aws iot with their new custom authentication beta which lets you pass auth data to your own custom backend iot authorizer via the username/password fields etc. I wasn't sure if mentioning specific cloud solutions made sense for this repo (?)

Reply:
I mean I am biased since I am at Microsoft... but removing the specific cloud provider from the sample seems like a nice thing to do. I'd like for this repo to remain company agnostic

Reply:
ok yeah I agree, the cloud providers can always reference in their own docs